src/runtime/particle-spec.ts

```diff
@@ -415,6 +415,20 @@ export class ParticleSpec {
     return !this.isolated;
   }
 
+  /**
```

Comment:
only somewhat related to this change - maybe rename the getters above `get isEgress(): boolean {`, `get isIsolated(): boolean {` - it might be more readable, now that we have egressType getter as well.

Reply:
Done